<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(181, 181, 181);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btn_0">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="btn_abs">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>abs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn_9">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btn_cos">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_ac">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: red;
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 15pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_7">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btn_6">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btn_mul">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string> × </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_4">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn_3">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_5">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btn_2">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="btn_c">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>⏎</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_po">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="btn_cot">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>cot</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" rowspan="2">
       <widget class="QPushButton" name="btn_eql">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>170</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_sum">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_fac">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>fac</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn_sin">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="btn_dot">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_8">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_1">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btn_sub">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btn_tan">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btn_div">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 255);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 12pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>➗</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_sqrt">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>sqrt</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="btn_rou">
        <property name="styleSheet">
         <string notr="true">border-width: 2px;
background-color: rgb(0, 255, 127);
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
        <property name="text">
         <string>rou</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLineEdit" name="text_line">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(84, 84, 84);
border-width: 2px;
border-radius: 20px;
border-color: black;
padding: 4px;
font: 20pt &quot;MV Boli&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
